src/test/java/StringTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StringTest {
+    // 요구사항 1
+    private List<Integer> split(String seq) {
+        return Arrays.stream(seq.split(","))
+                .map(String::trim)
+                .map(Integer::parseInt)
+                .collect(Collectors.toList());
+    }
+
+    @Test
+    void splitTest1() {
+        assertEquals(List.of(1, 2), split("1,2"));
```

Comment:
`assertj`의 `assertThat` 메소드를 사용하면 쉽게 테스트할 수 있습니다.
또한 제공하는 메소드 이름을 외우지 않고도 IDE를 통해 여러 기능들을 확인할 수 있어요

```suggestion
        Assertions.assertThat("1,2".split(",")).containsExactly("1", "2");
```